src/components/SearchBar/SearchBar.scss

```diff
@@ -0,0 +1,84 @@
+.searchBarContainer {
+  background-color: var(--color-fill-top-nav);
+  border-radius: 6px;
+  display: flex;
+  flex-direction: column;
+  height: fit-content;
+  max-height: 90vh;
+  min-height: 5em;
+  overflow-y: scroll;
```

Comment:
```suggestion
```

For what it's worth, this change looked better for me on Chrome/Safari/Firefox. But I'm also on MacOS 😅 

<img width="1154" alt="Screen Shot 2022-06-24 at 4 13 27 pm" src="https://user-images.githubusercontent.com/21051488/175473953-b51a1075-a953-4e74-9769-3fbe5d1a3e55.png">

Reply:
Oh, had no idea this was generating an outer scroll-bar 🤯